src/main.ts

```diff
@@ -0,0 +1,50 @@
+import * as core from '@actions/core';
+import * as tc from '@actions/tool-cache';
+import * as installer from './installer';
+import * as path from 'path';
+
+export async function run() {
+  try {
+    //
+    // versionSpec is optional.  If supplied, install / use from the tool cache
+    // If not supplied then problem matchers will still be setup.  Useful for self-hosted.
+    //
+    let versionSpec = core.getInput('go-version');
+
+    // stable will be true unless false is the exact input
+    // since getting unstable versions should be explicit
+    let stable = Boolean(core.getInput('stable') || 'true');
```

Comment:
from a node repl:
```
> Boolean('false')
true
```

in checkout, i did this: `(core.getInput('stable') || 'true').toUpperCase() === 'TRUE'`

Reply:
Will change.  And I'll add units / e2e around stable